labs/RivePlayer/samples/RivePlayer.Samples/RivePlayer.md

```diff
@@ -27,6 +27,25 @@ This library allows for control over Rive files with a high-level API for drivin
 
 ## Getting Started
 
+### Installing the package
+
+To add RivePlayer from Windows Toolkit Labs, you can use the NuGet Package Manager:
+- Add the `CommunityToolkit-Labs` package via the NuGet Package Manager in Visual Studio. [See more here](https://dev.azure.com/dotnet/CommunityToolkit/_artifacts/feed/CommunityToolkit-Labs/connect/visual%20studio)
+- **(Recommended: UWP)** Install the RivePlayer with the Package Manager Console. See more on the [artifacts page](https://dev.azure.com/dotnet/CommunityToolkit/_artifacts/feed/CommunityToolkit-Labs/NuGet/CommunityToolkit.Labs.Uwp.RivePlayer/overview/0.0.1)
+
+```
+// Replace with the appropriate version at the end
+Install-Package CommunityToolkit.Labs.Uwp.RivePlayer -version 0.0.1
```

Comment:
I have a PR up that bumps the version to 0.0.2. Maybe we should use that instead of 0.0.1?